### PR TITLE
PIM-3833: Count for items to mass edit

### PR DESCRIPTION
**Pull request by solivier**
| Q                    | A
| -------------------- | ---
| Bug fix?             | No
| New feature?         | Yes
| BC breaks?           | Yes
| CI currently passes? | Yes
| Tests pass?          | No
| Scenarios pass?      | No (But the one added for my code yes)
| Checkstyle issues?*  | No (not on my code)
| PMD issues?**        | No (not on my code)
| Changelog updated?   | No
| Fixed tickets        | PIM-3833
| DB schema updated?   | No
| Migration script?    | No
| Doc PR               | No

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php [hunk 1]
```diff
@@ -278,6 +283,11 @@ protected function getJobInstance($id, $checkStatus = true)
         return $jobInstance;
     }
 
+    /**
+     * @param $gridName
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php]: @param string $gridName

File: src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php [hunk 2]
```diff
@@ -290,6 +300,11 @@ protected function getItemsName($gridName)
         }
     }
 
+    /**
+     * @param $availableOperations
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Controller/MassEditActionController.php]: @param array $availableOperations ?

File: features/mass-action/select_products_to_mass_edit.feature [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  Scenario: Successfully count the number of mass-edited items when click on all visible products
+    Given I select all visible products
+    When I press mass-edit button
+    Then I should see "Mass Edit (10 products)"
+
+  Scenario: Successfully count the number of mass-edited items by select them one by one
+    Given I mass-edit products boot, shoe_1, shoe_14
+    Then I should see "Mass Edit (3 products)"
+
+  Scenario: mass edit on product with filters and select all action
```
Comment (grena) [inline features/mass-action/select_products_to_mass_edit.feature]: `Scenario: Successfully count the number of mass-edited items when using filters` ?

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/action/mass-action.js [hunk 4]
```diff
@@ -56,7 +56,14 @@ function(_, messenger, __, Modal, AbstractAction) {
                 values: idValues.join(',')
             };
 
+            if (1 === params.inset) {
```
Comment (grena) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/action/mass-action.js]: I think `if (selectionState.inset) {` is more readable for the future.

**Comment by grena** [comment]
GTM

**Pull request merged**